kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -73,13 +66,60 @@ public BigQueryRecordConverter(boolean shouldConvertDoubleSpecial) {
    */
   public Map<String, Object> convertRecord(SinkRecord kafkaConnectRecord) {
     Schema kafkaConnectSchema = kafkaConnectRecord.valueSchema();
+    Object kafkaConnectValue = kafkaConnectRecord.value();
+    if (kafkaConnectSchema == null) {
+      if (kafkaConnectValue instanceof Map) {
+        return (Map<String, Object>) convertSchemalessRecord(kafkaConnectValue);
+      }
+      throw new ConversionConnectException("Only Map objects supported in absence of schema for " +
+              "record conversion to BigQuery format.");
+    }
     if (kafkaConnectSchema.type() != Schema.Type.STRUCT) {
       throw new
           ConversionConnectException("Top-level Kafka Connect schema must be of type 'struct'");
     }
     return convertStruct(kafkaConnectRecord.value(), kafkaConnectSchema);
   }
 
+  private Object convertSchemalessRecord(Object value) {
+    if (value == null) {
+      return null;
+    }
+    if (value instanceof Double) {
+      return convertDouble((Double) value);
+    }
+    if (ClassUtils.isPrimitiveWrapper(value.getClass()) || value instanceof String) {
```

Comment:
is it really worth it to bring in a whole new dependency just to save some characters? This seems easy to replicate without needing ClassUtils.